doc/using/QuickStartGuide.rst

```diff
@@ -58,38 +58,38 @@ The `heartbeat` program
 
 .. code-block:: VHDL
 
-  entity hello_world is
-    port ( clk: out std_logic; )
-  end hearbeat;
-
-  architecture behaviour of hello_world is
-  begin
-    -- Clock process definition
-    clk_process: process
-    begin
-      clk <= '0';
-      wait for clk_period/2;
-      clk <= '1';
-      wait for clk_period/2;
-    end process;
-  end behaviour;
+   entity hello_world is
+      port ( clk: out std_logic; )
+   end hearbeat;
+
+   architecture behaviour of hello_world is
+   begin
+      -- Clock process definition
+      clk_process: process
+      begin
+         clk <= '0';
```

Comment:
Why has indentation changed?

Reply:
The first example of a code block I found was in the [Quick Start Guide](https://ghdl.readthedocs.io/en/latest/using/QuickStartGuide.html). It used a 3-space indent. Therefore, I standardized all other documents to use a 3-space indent inside code block examples.